contracts/src/libraries/Constants.sol

```diff
@@ -5,7 +5,55 @@ pragma solidity 0.8.20;
 address constant ETH = 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
 
 /// @dev The version of the contracts.
-string constant VERSION = "v1.0.11";
+string constant VERSION = "v1.0.12";
```

Comment:
I went ahead and updated this so that we can cut a new release once this PR and the audit changes land.